cypress/integration/swapPage.spec.ts

```diff
@@ -5,13 +5,68 @@ describe("The page for a swap", () => {
     cy.server();
     cy.route(
       "http://localhost:8000/swaps/rfc003/new-swap-id",
-      "fixture:swapSent.json"
+      "fixture:swap/swapSent.json"
     );
 
     cy.visit("/show_resource/swaps/rfc003/new-swap-id");
-    cy.get("[data-cy=swap-sent-card-header]").should("exist");
+    cy.get("[data-cy=swap-metadata-card]").should("exist");
     cy.get("[data-cy=alpha-asset-card-header]").should("exist");
     cy.get("[data-cy=beta-asset-card-header]").should("exist");
+    cy.get("[data-cy=swap-sent-card-header]").should("exist");
+  });
+
+  it("should display the action buttons in the right location for a newly received swap", () => {
```

Comment:
🙃 I found "in the right location" a little misleading because it has not really much to do with the location. I would probably just generally remove it. Note that there are several similar changes to be made in this file if you would like to change it. 
```suggestion
  it("should display the action buttons for a newly received swap", () => {
```

Reply:
I'm not against removing that phrase if it's confusing, but the tests actually do check whether buttons appear *in the right location*. For example,

```
cy.get("[data-cy=communication-card]").find("[data-cy=accept-button]");
```

which verifies that the accept button appears inside the communication card component and not just anywhere on the page.